node/src/command.rs

```diff
@@ -62,22 +76,29 @@ impl SubstrateCli for Cli {
 		load_spec(id)
 	}
 
-	fn native_runtime_version(_: &Box<dyn ChainSpec>) -> &'static RuntimeVersion {
-		&neumann_runtime::VERSION
+	// TODO: get the right runtime
+	fn native_runtime_version(
+		chain_spec: &Box<dyn sc_service::ChainSpec>,
+	) -> &'static RuntimeVersion {
+		if chain_spec.is_turing() {
```

Comment:
match better here for future?

Reply:
ya whenever we add OAK match will be better. I thought it looked weird bc it's 1 condition then `_` but I can change it